core/storage/base_model/gae_models_test.py

```diff
@@ -113,47 +113,48 @@ class TestVersionedModel(base_models.VersionedModel):
 
 class TestCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
     @classmethod
-    def _get_instance_id(cls, instance_id, version):  # pylint: disable=unused-argument
-        """A dummy function to mimic the actual _get_instance_id. In this case,
-        the actual id itself is the argument with this function just returning
-        the same value.
+    def _get_instance_id(cls, entity_id, version):
```

Comment:
If you're calling it target_entity_id in the superclass then do that here as well (and in the docstring). Ditto elsewhere.

Reply:
Done.